src/Core/src/ImageSources/Windows/ImageSourceServiceResult.cs

```diff
@@ -0,0 +1,33 @@
+﻿#nullable enable
+using System;
+using WImageSource = Microsoft.UI.Xaml.Media.ImageSource;
+
+namespace Microsoft.Maui
+{
+	public class ImageSourceServiceResult : IImageSourceServiceResult<WImageSource>
```

Comment:
Would it be useful to expose something from here indicating that it's been disposed? so then things using the result can check that and not use the result?

Reply:
Let me ad a flag anyway just to be safe.